packages/mainmenu-extension/src/index.ts

```diff
@@ -309,6 +322,26 @@ function createKernelMenu(app: JupyterLab, menu: KernelMenu): void {
     execute: Private.delegateExecute(app, menu.kernelUsers, 'shutdownKernel')
   });
 
+  commands.addCommand(CommandIDs.shutdownAllKernels, {
+    label: 'Shutdown All Kernels',
```

Comment:
Now that there is a prompt for user action, can we add an ellipsis to the label?